setup.cfg

```diff
@@ -41,6 +41,7 @@ install_requires =
     pandas
     ray>=1.0
     torch
+    functorch
```

Comment:
functorch has had some installation compatibility issues in the past. Even the previous release wasn't working on certain Ubuntu versions when installed via `pip install functorch`, although I believe this is now fixed. I'm not sure there is anything we can do about this but it's worth being aware of it.